velox/functions/sparksql/tests/ArithmeticTest.cpp

```diff
@@ -436,6 +437,70 @@ TEST_F(ArithmeticTest, isNanDouble) {
   EXPECT_EQ(false, isNan(std::nullopt));
 }
 
+TEST_F(ArithmeticTest, normalizeNanFloat) {
+  const auto normalizeNan = [&](std::optional<float> a) {
```

Comment:
remove std::optional; we don't need to pass std::nullopt

Reply:
Changed, thanks.